stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -79,6 +79,66 @@ var result = websub:startHub(new http:Listener(8080), hubConfiguration = hubConf
 
 Any subscriptions added at the hub will now be available even when the hub is restarted.
 
+##### Enabling Basic Auth support for the hub
+
+The Ballerina WebSub Hub can be secured using basic authentication by enforcing authorization. Hub listener and the 
+Hub service need AuthProvider and authConfig configurations respectively. The auth provider stores username, password
+for authentication and scopes for authorization. User can specify AuthProvider as follows pass it along with the 
+hubListenerConfig when creating hub.
+
+``` ballerina
+http:AuthProvider basicAuthProvider = {
+    scheme: "basic",
+    authStoreProvider: "config"
+};
+
+http:ServiceEndpointConfiguration hubListenerConfig = {
+    authProviders: [basicAuthProvider],
+    secureSocket: {
+        keyStore: {
+            path: "${ballerina.home}/bre/security/ballerinaKeystore.p12",
+            password: "ballerina"
+        },
+        trustStore: {
```

Comment:
Is there any particular reason for configuring trustStore? Wouldn't it be enough to configure only the keystore?